### PR TITLE
Retrieves and updates data in DynamoDB when email notifications are sent

### PR DESCRIPTION
**Pull request by joshmsamuels**
Sample DynamoDB schema
Screen Shot 2020-07-31 at 5 12 40 PM

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,33 +1,60 @@
 // app.js
 
-const { exec } = require('child_process')
+const {
+  exec
+} = require('child_process')
```
Comment (jayellul) [inline app.js:5]: What is going on here with our prettier lol

File: app.js [hunk 2]
```diff
@@ -192,34 +251,150 @@ async function sendEmail(courseMap, recipients) {
...
+}
+
+/**
+ * Queries dynamo db to find the user's to notify for each course
+ * 
+ * @param {Array} courseCodes - Array of course codes of available courses
+ * @returns {Object} - Keys correspond to the course codes and the value 
+ * is an array of emails to send the course to
+ */
+async function getEmailsCourseCodesFromDyanmo(courseCodes) {
```
Comment (jayellul) [inline app.js:300]: Super NIT: Dynamo spelt wrong in the function declaration

**Review by jayellul — APPROVED** [review]
LGTM 👍

File: app.js (hunk 1 above)
Comment (joshmsamuels) [reply app.js:5]: not sure LMAO but we can look into it in another PR

**Pull request closed without merging**